src/edu/cscc/mvc/CreateRental.java

```diff
@@ -0,0 +1,55 @@
+package edu.cscc.mvc;
+
+import edu.cscc.mvc.framework.ApplicationView;
+import edu.cscc.mvc.framework.MVCContext;
+
+import java.util.*;
+
+public class CreateRental extends ApplicationView {
+    private UUID rentalID;
+    /**
+     * Sets the context and configures it to exit by default
+     * unless a route is specified.
+     *
+     * @param context The {@link MVCContext}.
+     */
+    public CreateRental(MVCContext context, UUID rentalID) {
+        super(context);
+        this.rentalID = rentalID;
+    }
+
+    @Override
+    public void show() {
+        Scanner scanner = new Scanner(System.in);
+        Map params = new HashMap<>();
+        try {
+            params.put("rentalID", rentalID);
+
+            System.out.println("Create a Rental:\n****************");
+
+            System.out.println("Name: ");
+            String rentalName = scanner.nextLine();
+            params.put("rentalName", rentalName);
+
+            System.out.println("Format:\n" + "\t1. DVD\n" + "\t2. VHS\n" + "\t3. BLU_RAY\n" + "Choice: ");
```

Comment:
This works, but it would be cleaner to use separate `System.out.println()` calls.